services/pub/pub-publisher.tester.js

```diff
@@ -0,0 +1,26 @@
+import { ServiceTester } from '../tester.js'
+export const t = new ServiceTester({
+  id: 'PubPublisher',
+  title: 'Pub Publisher',
+  pathPrefix: '/pub',
+})
```

Comment:
Similarly as the class export, we prefer and encourage the model documented in our [tutorial on service tests](https://github.com/badges/shields/blob/master/doc/service-tests.md#1-boilerplate) and only fall back to this more verbose model when strictly necessary, such as when the corresponding *.service.js file has multiple exports

```suggestion
import { createServiceTester } from '../tester.js'
export const t = await createServiceTester()
```

Reply:
Gotcha, thanks - will update this.

Reply:
Hmm - when I make this change, my tests fail - `404`'ing against the server - and it doesn't look like the service itself is registered.

Reply:
Sorry, I forgot to mention that as part of this you can and will need to remove the `/publisher` path in your tests too since the test registration helper handles the full path automatically